lib/fileHandler/index.js

```diff
@@ -276,14 +287,14 @@ module.exports = function (options) {
 						return reject(formatUploadRejectObject(fileUploadError));
 					}
 
-					const twentyFourHourPeriod = moment().add(1, 'days');
+					const sixHourPeriod = moment().add(6, 'hours');
```

Comment:
Why is this being defined twice? 
Looks like the s3 upload function that we can call on line 158 and 169 are very similar, couldn't we turn this into a function rather than repeat the same code twice?